recipes/botan/all/conanfile.py

```diff
@@ -312,6 +312,10 @@ def _configure_cmd(self):
         botan_extra_cxx_flags = []
         build_flags = []
 
+        if Version(self.version) >= '3.7' and is_msvc(self) and Version(self.settings.compiler.version) < '194':
+            # https://github.com/conan-io/conan-center-index/pull/26531#issuecomment-2636504872
+            build_flags.append('--disable-modules=bitvector')
```

Comment:
Note that the bitvector is mostly a helper structure for the post-quantum algorithm "Classic McEliece". By disabling bitvector you also disable this algorithm.

```suggestion
        if Version(self.version) >= '3.7' and is_msvc(self) and Version(self.settings.compiler.version) < '194':
            # https://github.com/conan-io/conan-center-index/pull/26531#issuecomment-2636504872
            # Note that this disables support for Classic McEliece
            build_flags.append('--disable-modules=bitvector')
```